### DISCUSSION TITLE
[Request] Pyload

### DISCUSSION THREAD

**Discussion by Brenders** [tteck/Proxmox#2221] [Ideas]
Hi would it be possible to have Pyload implemented in LXC.
the official site is this one https://pyload.net/ while the github page is this one https://github.com/pyload/pyload .
Pyload is Free and Open Source download manager written in Python and designed to be extremely lightweight, easily extensible and fully manageable via web is perfect to implement LXC because we can run it in containers.

**Comment by dasaferi** [comment]
@tteck have you tried creating an lxc package using this guide https://github.com/pyload/pyload?tab=readme-ov-file#advanced-installation from what I have noticed the process is very simple because it requires no dependencies outside of python.

**Reply by tteck** [reply]
I have not.

**Comment by Brenders** [comment]
@tteck did you have time to create the lxc container? Could you link me how I can do it and if I can I'll send a pull request

**Reply by tteck** [reply]
Have you tried the Aria2 LXC?

**Reply by Brenders** [reply]
I've tried it for a long time and it's not the same as pyload because I can't log in to my account but I have to do complicated things to make it go with the account of the site I need to download files, the only thing I've noticed is that aria2 works well with torrents that I'm not interested in.
